packages/svelte/src/compiler/utils/sanitize_template_string.js

```diff
@@ -1,7 +1,12 @@
+const BOM_CHARACTER = String.fromCharCode(65279);
+
 /**
  * @param {string} str
  * @returns {string}
  */
 export function sanitize_template_string(str) {
-	return str.replace(/(`|\${|\\)/g, '\\$1');
+	return str
+		.replace(/(`|\${|\\)/g, '\\$1')
```

Comment:
Is this the right place? BOM can only appear at the very beginning of the file. I think we should do it at the very beginning of the parser instead.

I'm still a bit confused why this only turns up now though. Would've thought that tooling that feeds us this stuff handles it

Reply:
Ironically, I found a better approach that does just that!